tests/sphinxext/reference_results/workchain.xml

```diff
@@ -70,25 +70,37 @@ finalize</literal_block></paragraph>
         </desc>
         <paragraph>The command is also hooked into <literal>sphinx.ext.autodoc</literal>, so AiiDA processes will be properly documented using <literal>.. automodule::</literal> as well.</paragraph>
         <target ids="module-demo_workchain" ismod="True"></target>
-        <index entries="['single',\ 'demo_workchain\ (module)',\ 'module-demo_workchain',\ '',\ None]"></index>
+        <index entries="['pair',\ 'module;\ demo_workchain',\ 'module-demo_workchain',\ '',\ None]"></index>
         <paragraph>This module defines an example workchain for the aiida-workchain documentation directive.</paragraph>
-        <index entries="['single',\ 'DemoWorkChain\ (class\ in\ demo_workchain)',\ 'demo_workchain.DemoWorkChain',\ '',\ None]"></index>
         <desc desctype="class" domain="py" noindex="False" objtype="class">
-            <desc_signature class="" first="False" fullname="DemoWorkChain" ids="demo_workchain.DemoWorkChain" module="demo_workchain" names="demo_workchain.DemoWorkChain"><desc_annotation xml:space="preserve">class </desc_annotation><desc_addname xml:space="preserve">demo_workchain.</desc_addname><desc_name xml:space="preserve">DemoWorkChain</desc_name><desc_parameterlist xml:space="preserve"><desc_parameter xml:space="preserve">inputs=None</desc_parameter><desc_parameter xml:space="preserve">logger=None</desc_parameter><desc_parameter xml:space="preserve">runner=None</desc_parameter><desc_parameter xml:space="preserve">enable_persistence=True</desc_parameter></desc_parameterlist></desc_signature>
+            <desc_signature first="False" fullname="WorkChain"><desc_annotation xml:space="preserve">workchain</desc_annotation><desc_addname xml:space="preserve">demo_workchain.</desc_addname><desc_name xml:space="preserve">DemoWorkChain</desc_name></desc_signature>
```

Comment:
Looks like the output changed here (e.g. from `DemoWorkChain` to `WorkChain`), is this intended?

Reply:
removed this change